vortex-array/src/data.rs

```diff
@@ -123,7 +123,7 @@ impl From<ArrayData> for Array {
 
 impl Statistics for ArrayData {
     fn get(&self, stat: Stat) -> Option<Scalar> {
-        self.stats_map.read().unwrap().get(stat).cloned()
+        self.stats_map.read().ok()?.get(stat).cloned()
```

Comment:
Ok, I know you're trying to make this nice but since this is a lock the meaning of converting this error into empty is a bit misleading. This will only error if a thread holding the lock had died and had not released it. All to say... this likely should be one exception